### PR TITLE
add docker-compose and makefile

### PR DESCRIPTION
**Pull request by paolino**
- [x] add code to self-host a github-runner

### CODE REVIEW

**Review by paweljakubas — APPROVED** [review]

File: scripts/github-runner/setup.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+## How to prepare a self-hosted runner for this repo
+
+- get your `github-token` from [https://github.com/settings/tokens](https://github.com/settings/tokens), allowing access to the repo and to the workflow (to be asessed)
```
Comment (piotr-iohk) [inline scripts/github-runner/setup.md]:
```suggestion
- get your `github-token` from [https://github.com/settings/tokens](https://github.com/settings/tokens), allowing access to the repo and to the workflow (to be assessed)
```

**Review by piotr-iohk — APPROVED** [review]
:+1:

**Comment by paolino** [comment]
bors try

**Comment by paolino** [comment]
deemed not a good idea

**Pull request closed without merging**